codegen/codegen.go

```diff
@@ -19,6 +19,8 @@ func Generate(cfg Config) error {
 		return err
 	}
 
+	DefaultPluginRegistry.Execute(&cfg, cfg.schema)
```

Comment:
maybe rather than init/globals this should just be on config, not exposed to yaml:

```
    plugins []Plugin "yaml:\"-\""
```

and in main we load up the default ones, allowing custom binaries with new plugins to be easily created.